presentation.md

```diff
@@ -7,5 +7,45 @@ abstract: |
   Lorem ipsum dolor sit amet
 ---
 
-# Document Title
+# Źródła danych
 
+To było pierwszym sporym wyzwaniem w naszym projekcie, ponieważ szczególnie na
+początku pandemii dostęp do szczegółowych danych nie był tak prosty, ponieważ:
+
+- na stronie ministerstwa publikowane były wyłącznie dane z danego dnia.
+- publicznie dostępne dane dotyczące wielu krajów nie zawierały podziału na
+  województwa.
+
+Na szczęście udało nam się znaleźć inne źródła, tj. strony tworzone przez ludzi,
+którzy codziennie sami zbierali te dane i udostępniali je w formie wykresów.
+Przykładem takiej strony jest koronawirusunas.pl . Dzięki danym na niej zawartym
+udało nam się przeprowadzić.
+
+Do głównych źródeł danych zaliczyć możemy:
+
+- IHME
+- koronawirusunas.pl
+- policja.pl
```

Comment:
You can split slides with 
```md
Lorem

---

ipsum
```
This slide is a bit too long

Reply:
Done.